src/MessagingInput/MessagingInput.less

```diff
@@ -4,43 +4,50 @@
   position: relative;
 }
 
-.MessagingInput--ShowUploadAttachmentButton .FileInput {
-  height: @size_2xl;
-  .padding--none();
-  border: none;
+.MessagingInput--InnerContainer {
+  align-items: flex-end;
+  width: 100%;
+}
 
-  // Hide FileInput after file has been selected and uploading has begun
-  &.FileInput--AttachmentIsUploading {
-    display: none;
-  }
+.MessagingInput--InnerContainer--Content {
+  width: 100%;
+  .margin--right--xs();
+  overflow-y: auto;
+}
 
-  .FileInput--Icon {
-    margin: 0.25rem 0.5rem 0.25rem 0.75rem;
-    .borderRadius--l();
+// ----- Info that shows above reply/text area
 
-    &:hover {
-      fill: #f5f8fd;
-    }
-  }
+.MessagingInput--TopInfo--Container {
+  .margin--bottom--2xs();
 }
 
-.MessagingInput--Checkbox {
+.MessagingInput--TopInfo--Checkbox {
```

Comment:
The only changes in this file are:
1. Renamed items that are above the MessagingInput to be prefixed with `TopInfo` - Checkbox and label. Feel free to give advice on a better name! 
2. Reorganize all the classes so they are separated by relevant category - i.e. moved all attachments related classes together
3. Add separator comments

Reply:
⬆️  very helpful context, thank you!

I think the name `TopInfo` implies something about the layout and styling (that it's on top) that could be changed instead of naming the group of elements by what role they play on the page. The "`Top`" part also feels like info that should be captured in the styling, not in the name. What do you think of these possible ideas?

`HelperText`
`DraftHelperText`
`InfoText`

All of those include "`Text`" and I guess, to me, that feels more specific than "`Info`" because info could be numerical, graphical, textual, etc.

Reply:
Thanks for raising those options! Since this area currently has the two pieces of information seen here - "Send an message/announcement" and the checkbox - I felt I needed something general enough to describe whatever is above the messaging input because I didn't see any similarity between those two items. And since the checkbox is functional beyond just being text, im not sure if `Text` captures that :D They may both relate to the draft in a way or helpers, but still a little uncertain

![Screen Shot 2021-06-23 at 12 06 21 PM](https://user-images.githubusercontent.com/26425483/123706980-ddb1cb80-d81d-11eb-89a9-b1e9b10d673a.png)